src/modules/accommodations/accommodations.controller.ts

```diff
@@ -23,4 +26,18 @@ export class AccommodationsController {
       limit: searchQueryParams.limit,
     };
   }
+
+  @Post(':id/like')
```

Comment:
I think it'd be better to create a separate module for wishlist...

Reply:
I do not think so, there is only one endpoint here for wishlist. It would be extra work if we implement wishlist on separate module

Reply:
we have category and amenities modules, right?  if you do this in the accommodation module, it would be irrelevant because the wishlist belongs to user

Reply:
actually I did it on user module, but Muhammadjon teacher said to implement it on accommodation, not in user. So I changed it to accommodation